GetUserDataProcessingActivity/handler.ts

```diff
@@ -0,0 +1,212 @@
+/**
```

Comment:
why do we need this activity ? it looks like it's a convenience method used only by the cli